### DISCUSSION TITLE
Wifi Component bugged in Multiplayer?

### DISCUSSION THREAD

**Discussion by GenderedHuman** [FakeFishGames/Barotrauma#14136] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Me and a friend tried using the wIfi component connected to a terminal to display our radio text in game on any channel and it works, until another person talks in the radio channel and it stop sending our text to the terminal.
We change channels in the wifi component and talk in that one and it'll work again until a the second person talks in it again and it'll once again stop sending our text to the terminal.
We are playing on Vanilla and have linking wifi components to chat enabled. 

### Reproduction steps

1. get a wifi component and terminal and set the wifi component it to any channel (and have another player)
2. wire the wifi components "signal out" to a terminals "signal in"
3. talk in the radio channel you set your wifi component to and It'll show up in the terminal
4. have another person talk in the radio channel
5. now it'll stop showing up in the terminal until you switch the wifi components channel

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested a fix for this.
Closing discussion as resolved.
